source/guide-to-being-a-mentor.md

```diff
@@ -15,7 +15,7 @@ submit ideas via JIRA (if your project does not use JIRA you can [use the Comdev
 * Add an issue to JIRA (if your project doesn't use JIRA contact dev@community.apache.org)
```

Comment:
1. `if your project does not use JIRA you can [use the Comdev Issue Tracker For GSoC Tasks](use-the-comdev-issue-tracker-for-gsoc-tasks.html`
2. `if your project doesn't use JIRA contact dev@community.apache.org`

Hi, it looks like these statements above will confuse users since the condition is the same, but there are two solutions for me to choose? 
If we want those mentors whose community dose not use JIRA to use `Comdev Issue Tracker`, maybe it is unnecessary to tell them to ask this question in dev@ml? (I guess if so, we will receive many emails with the same questions.  😃 )
Also, a similar statement appears in `gsoc.md`.

Reply:
Thanks @tristaZero,

could you please check the new version? is it more clear?